discovery-provider/src/tasks/users.py

```diff
@@ -404,10 +403,6 @@ def update_user_associated_wallets(
                     blockhash=user_record.blockhash,
                 )
                 session.add(associated_wallet_entry)
-
```

Comment:
If we don't have this, then we won't enqueue initial refreshes for newly associated wallets